fetch.bs

```diff
@@ -2690,16 +2693,104 @@ functionality.
 <dfn export for="fetch record" id=concept-fetch-record-fetch>controller</dfn> (a
 <a for=/>fetch controller</a> or null).
 
+<p>A <dfn export>deferred fetch record</dfn> is a <a for=/>struct</a> used to maintain state needed
+to invoke a fetch at a later time, e.g., when a <code>Document</code> is unloaded or backgrounded. It
```

Comment:
Is this equivalent to being bfcached? (Sorry not sure if this is the standard term of it)

Reply:
Yea, it's when it's BFCached

Reply:
Maybe refer to BFCache's [spec](https://w3ctag.github.io/bfcache-guide/#intro) and replace `backgrounded` with the term `non-fully active`?

> After a user navigates away from a document, the document might be cached in a non-[fully active](https://html.spec.whatwg.org/multipage/document-sequences.html#fully-active) state